middlewares.go

```diff
@@ -58,7 +58,25 @@ func JwtMiddleware(key string) negroni.Handler {
 		},
 		SigningMethod: jwt.SigningMethodHS256,
 	})
-	return negroni.HandlerFunc(jwtMiddleware.HandlerWithNext)
+	// return negroni.HandlerFunc(jwtMiddleware.HandlerWithNext)
+	return negroni.HandlerFunc(func(w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
```

Comment:
extract this function into a new function with signature 
```go
func(w http.ResponseWriter, r *http.Request, next http.HandlerFunc)
 ``` 
to make this middleware testable and mockable